### ISSUE TITLE
NTR: atrophic pituitary gland

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4225]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Atrophic pituitary gland

### 2. Synonyms
Atrophic pituitary
Pituitary gland atrophy

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Partial or complete wasting (loss) of the pituitary gland.
Source: DOI: 10.1530/endoabs.44.EP82

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0012503 'Abnormality of the pituitary gland'
